src/components/QueryCondition.vue

```diff
@@ -91,9 +91,10 @@ export default Vue.extend( {
 				if ( selectedProperty === null ) {
 					this.$store.dispatch( 'unsetProperty', this.conditionIndex );
 					return;
+				} else if ( selectedProperty.datatype === 'wikibase-item' ) {
```

Comment:
I don't think we need this `else` as the previous `if` always returns

Reply:
I wasn't sure if we needed it either.